File: Week 6 - Python/mario/mario.py
import cs50


def main():
    # Promt user
    while True:
        i = cs50.get_int("Height: ")
        if i > 0 and i < 9:
            break

    # Call function
    pyramid(i, i)


def pyramid(i, j):
    if i > 1:
        pyramid(i - 1, j)
    print(" " * (j - i), end="")
    print('#' * i, end="")
    print("  ", end="")
    print('#' * i)


# call main
main()

File: Week 1 - C Programming/mario/mario.c
 8#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //declare elements of the game
    int height;
    int h = 0;

    //prompt for height (between 1 and 8)
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    //create the pyramids
    while (h < height)
    {
        //create space between the first pyramid and the left screen margin
        int space1 = 0;
        while (space1 < height - h - 1)
        {
            space1++;
            printf(" ");
        }

        //create the first pyramid
        int hash1 = 0;
        while (hash1 < h + 1)
        {
            hash1++;
            printf("#");
        }

        //create space between two pyramids
        printf("  ");

        //create the second pyramid
        int hash2 = 0;
        while (hash2 < h + 1)
        {
            hash2++;
            printf("#");
        }

        printf("\n");
        h++;
    }


}
